.isort.cfg

```diff
@@ -1,4 +0,0 @@
-[settings]
-profile=black
-known_first_party=great_expectations,importlib_metadata # Added here because seed-isort-config does not recognize it
-known_third_party=IPython,_pytest,altair,black,boto3,botocore,click,dateutil,freezegun,ipywidgets,jinja2,jsonpatch,jsonschema,marshmallow,mistune,mock,moto,nbconvert,nbformat,numpy,pandas,pkg_resources,pyparsing,pytest,requests,requirements,ruamel,scipy,setuptools,sphinx,sqlalchemy,tzlocal
```

Comment:
I have left these settings out of the `pyproject.toml` config.
Changes as of `isort 5` make these no longer necessary. 

Screenshot below showing isort being run with these setting outside of the virtual environment without resulting in any sorting changes/regressions.

![image](https://user-images.githubusercontent.com/13108583/179771447-196c8554-0c52-4779-82a4-b3c52103b4a4.png)